src/cmd/run_force_mode.rs

```diff
@@ -0,0 +1,100 @@
+use async_trait::async_trait;
+use miette::{Result, miette};
+use multitool_sdk::models::{RolloutStateStatus, RolloutStateType, RolloutStatus};
+use tokio::time::{Duration, sleep};
+use tracing::trace;
+
+use crate::WholePercent;
+use crate::adapters::{BackendClient, BoxedIngress, BoxedMonitor, BoxedPlatform, RolloutMetadata};
+
+use crate::cmd::run::DeploymentMode;
+
+/// Force deployment mode - bypasses canary analysis and immediately deploys at 100%
+pub struct ForceMode;
+
+const POLL_INTERVAL: Duration = Duration::from_secs(5);
+
+#[async_trait]
+impl DeploymentMode for ForceMode {
```

Comment:
Can you add a doc comment here explaining the steps that force mode takes, and how we choose not to boot the subsystems for simplicity?